dashboard/config/routes.rb

```diff
@@ -14,7 +14,6 @@
   get '/terms-and-privacy', to: 'home#terms_and_privacy'
   get '/dashboardapi/terms-and-privacy', to: "home#terms_and_privacy"
   get '/dashboardapi/teacher-announcements', to: "home#teacher_announcements"
-  get '/dashboardapi/hoc-courses-narrow', to: "home#hoc_courses_narrow"
```

Comment:
I _believe_ this route is extraneous for a few reasons: 

1. The only place that would use this route is [hoc-overview.js](https://github.com/code-dot-org/code-dot-org/blob/47379314467b1bad950908600ef9473b893dd8e7/pegasus/sites.v3/code.org/public/js/hoc-overview.js#L2), which calls `/dashboardapi/hoc_courses_narrow` (underscores rather than hyphens).
2. There is no `hoc_courses_narrow` method in `HomeController`
3. Removing this route still works as expected

Maybe I'm missing something, so please double-check me here!